stage0/src/Init/Lean/Meta/Tactic/Util.lean

```diff
@@ -9,8 +9,8 @@ import Init.Lean.Meta.Basic
 namespace Lean
 namespace Meta
 
-def mkFreshExprSyntheticOpaqueMVar (type : Expr) (userName : Name := Name.anonymous) : MetaM Expr :=
-mkFreshExprMVar type userName MetavarKind.syntheticOpaque
+def mkFreshExprSyntheticMVar (type : Expr) (userName : Name := Name.anonymous) : MetaM Expr :=
```

Comment:
This change is undoing one of my fixes. Moreover, it is changing a file at `Tactic` in a PR about `IO`.